examples/examples_test.go

```diff
@@ -115,6 +115,14 @@ func TestExampleObjectSchemas(t *testing.T) {
 			"redis-master-service":   &api.Service{},
 			"redis-slave-service":    &api.Service{},
 		},
+		"../examples/guestbook/guestbook_ns": {
```

Comment:
I have added it in a new folder called guestbook_ns.
If changes look good , I will merge with guestbook folder